pkg/utils/pods.go

```diff
@@ -10,8 +10,15 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-// GetFenceAgentsRemediationPod fetches the FAR pod based on FAR's label and namespace
+// GetFenceAgentsRemediationPod fetches the first pod that match to FAR's label and namespace
 func GetFenceAgentsRemediationPod(r client.Reader) (*corev1.Pod, error) {
+	podlsit, err := GetFARPodList(r)
+	return &podlsit.Items[0], err
```

Comment:
what about returning the pod which is in `Running` phase?

Reply:
Sounds better :)